examples/algo-example/src/app/algo.erc20.example.ts

```diff
@@ -0,0 +1,53 @@
+import { TatumAlgoSDK } from '@tatumio/algo'
+import { Currency, TransactionHash } from '@tatumio/api-client'
+
+const algoSDK = TatumAlgoSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
+
+// @TODO: remove ALGO from openApi - MINT ERC20 section
+
+export async function algoErc20Example() {
```

Comment:
for algorand, this is called ASA - rename the methods and tests to ASA, so its Algorand specific